blockly/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
 
 
 sourceSets.main.java.srcDirs = ['src/']
+sourceSets.main.resources.srcDirs += {project(':dungeon').sourceSets.main.resources.srcDirs}
```

Comment:
hat diese änderung was mit dem pr zu tun?

Reply:
hmm, ich denke, nicht unmittelbar.

hier geht es ja nur darum, dass es auch mit leerzeichen, etc. im pfad funktioniert ... nicht, das alles "spielbar" ist.

also wieder raus?

Reply:
@tgrothe langsam werde ich grummelig! wie oft noch. das thema dieses pr heisst "leerzeichen und umlaute in den pfaden erlauben". diese änderung an der build.gradle hat damit imho nicht mal ansatzweise zu tun. das gehört zu #1426!

Reply:
dieser pr ist schon bei 170+ kommentaren ... und ein ende ist nicht abzusehen. warum?!!!

Reply:
> dieser pr ist schon bei 170+ kommentaren ... und ein ende ist nicht abzusehen. warum?!!!

ich dachte, dieser pr wäre jetzt fast fertig?